controllers/apply/form-router-next/index.js

```diff
@@ -136,11 +137,12 @@ function initFormRouter({
      * Require active user past this point
      */
     router.use(
+        noStore,
+        requireActiveUser,
         handleMultipartFormData,
-        csrfProtection,
         injectCopy('applyNext'),
         setCommonLocals,
-        requireActiveUser
+        csurf()
```

Comment:
Move `requireActiveUser` handler up in the chain and initialise `csurf` later. This change ensures we don't throw an error if posting to a step after the session has expired.